lighthouse-core/audits/byte-efficiency/uses-webp-images.js

```diff
@@ -21,10 +21,10 @@ class UsesWebPImages extends ByteEfficiencyAudit {
   static get meta() {
     return {
       name: 'uses-webp-images',
-      description: 'Serve images as WebP',
+      description: 'Serve images in superior formats',
       informative: true,
-      helpText: 'WebP provides better lossy and lossless compression than PNG or JPEG, ' +
-        'which means faster downloads and less data consumption. ' +
+      helpText: 'Image formats like JPEG 2000, JPEG XR, and WebP often provide better ' +
```

Comment:
Do you want to mention something here about different browser support for each, or nah?

Reply:
Nah we have a policy to keep the details in the docs